week2/yeonwook/index.css

```
@@ -0,0 +1,152 @@
+button{
+    background: none;
+    border: none;
+    padding: 0;
+    cursor: pointer;
+    outline: inherit;
+}
+
+.uppersection{
+    position: relative;
+}
+
+.upperheader{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding: 12px 16px;
+    box-sizing: border-box;
+    border-bottom: 1px solid #eee;
+}
+
+.upperheader .utility{
+    padding : 8px;
+}
+
+.upperheader .town{
+    color : #000000;
+    font-family: Inter;
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 28px;
+    text-align: left;
+    padding-right: 6px;
+}
+
+.contents{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    align-items: center;
+    box-sizing: border-box;
+    border-bottom: 1px solid #eee;
+    padding-bottom: 80px;
+}
+
+.contents .itembutton{
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    border-bottom: 1px solid #eee;
+    padding: 20px;
+}
+
+.itemdescription{
+    display: flex;
+    flex-direction: row;
+}
+
+.itemcontent{
+    min-width: 100px;
+    display: flex;
+    flex-direction: column;
+    padding: 0 16px;
+}
+
+.itemcontent .itemtitle{
+    color : #000;
+    font-family: Inter;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 20px;
+    text-align: left;
+    margin-bottom : 6px;
+}
+
+.itemcontent .itemprice{
+    color:#ff7e36;
+    font-family: Inter;
+    font-size: 15px;
+    font-weight: 700;
+    line-height: 22px;
+    text-align: left;
+    margin-top : 6px;  
+}
+
+.details{
+    color : #8c8c8c;
+    font-family: Inter;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 16px;
+    text-align: left;
+}
+
+.upper_likesandchats{
+    display: flex;
+    align-items: center;
+    flex-direction: column-reverse;
+}
+
+.likesandchats{
+    padding-top: 90px;
+    display: flex;
+    align-items: center;
+}
+
+.iteminfo{
+    margin:3px
+}
+
+.mymenu{
+    display: flex;
+    justify-content: space-around;
+    position: fixed;
+    bottom: 0px;
+    width: 100%;
+    background-color:#fff;
+    padding-top: 10px;
+    padding-bottom: 20px;
```

```suggestion
    padding: 10px 0;
```
위아래 패딩을 다르게 준 이유가 있나요?